artemis-core-client/src/main/java/org/apache/activemq/artemis/api/core/jgroups/JChannelWrapper.java

```diff
@@ -39,7 +40,7 @@
    final List<JGroupsReceiver> receivers = new ArrayList<>();
    private final JChannelManager manager;
 
-   public JChannelWrapper(JChannelManager manager, final String channelName, JChannel channel) throws Exception {
```

Comment:
I don't see why you would need to make this change here? can you place the final back?